vcx/libvcx/tests/logging.rs

```diff
@@ -24,7 +26,7 @@ mod log_tests {
     use indy::future::Future;
 
     static mut COUNT: u32 = 0;
-    extern fn custom_log(_context: *const c_void,
```

Comment:
I suggest reverting these changes. 
We already fixed indy-sys to use `libc` types.